packages/react-components/src/components/SendBox.tsx

```diff
@@ -307,19 +309,28 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
     return props.onRenderFileUploads ? (
       props.onRenderFileUploads()
     ) : (
-      <Stack className={attachmentUploadCardsStyles}>
-        <_AttachmentUploadCards
-          activeFileUploads={activeFileUploads}
-          onCancelFileUpload={props.onCancelFileUpload}
-          strings={{
-            removeAttachment: props.strings?.removeAttachment ?? localeStrings.removeAttachment,
-            uploading: props.strings?.uploading ?? localeStrings.uploading,
-            uploadCompleted: props.strings?.uploadCompleted ?? localeStrings.uploadCompleted
-          }}
-        />
-      </Stack>
+      <FluentV9ThemeProvider v8Theme={theme}>
```

Comment:
Which of the fluent components in use here are from v9?

Reply:
AttachmentUploadCards inside of it need the styling